README.md

```diff
@@ -6,46 +6,286 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
-This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
+Git Machete plugin is a robust tool that simplifies your git related workflow.
+It's a great complement to the JetBrains products' built-in version control system.
+The "bird's eye view" provided by Git Machete makes merges/rebases/push/pulls hassle-free
+even when multiple branches are present in the repository
+(master/develop, your topic branches, teammate's branches checked out for review, etc.).
+A look at a Git Machete tab gives an instant answer to the question
+"what branches are in this repository, what is going to be merged (rebased/pushed/pulled) and to what?".
 
-This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
+With this plugin, you can simply maintain small, focused pull requests that are easy to review.
 
-![](docs/sample-workflow.gif)
+**It is compatible with all JetBrains products except Android Studio.
+The minimum required version is 2020.1**.
 
+![](docs/sample_workflow.gif)
 
-## Logging
+See also [git-machete](https://github.com/VirtusLab/git-machete#git-machete) &mdash; a CLI version of this plugin.
 
-SLF4J logging in this plugin has the following categories:
 
+## Installing
+
+### Install from JetBrains Marketplace
+
+This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete). <br/>
+To install this plugin go to `File` -> `Settings` -> `Plugins` in your IDE,
+then make sure you are on `Marketplace` tab (not `Installed`), in search box type `Git Machete` and click `Install`.
+After installation depending on IDE version restart may be required.
+In that case, just click `Restart IDE` and confirm that action in a message box.
+
+
+## How it works
+
+Git Machete IntelliJ Plugin is a port of a great console tool &mdash; [git-machete](https://github.com/VirtusLab/git-machete#git-machete), into an IntelliJ plugin.
+Git Machete can help you manage your repository branches and keep it in sync with each other and its counterparts on a remote repository.
+Let's see how this plugin can help you:
```

Comment:
Maybe remove this sentence coz the narrative doesn't continue into the `how this plugin can help you` things afterwards